content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md

```diff
@@ -79,15 +79,10 @@ Click on (+) for registering a storage backend
 
 and then provide valid region, endpoint, Access and Secret Key parameters
 
-##### Note:
-For cloud vendors like AWS, Azure, Huawei and GCP remote cloud buckets need not be created manually by user prior to backend registration. Remote cloud buckets can be created from SODA for these cloud vendors
+##### Register a cloud backends as follows
```

Comment:
nit: Change the heading level to `###` to match the subsequent heading levels.
For ex: The next heading is  `### Create bucket and upload the object`.